src/component/ProductContainer/ProductContainer.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/destructuring-assignment */
+import React, { Component } from 'react';
+import '../../component/ProductContainer/ProductContainer.scss';
+import '../ProductContainer/ProductContainer.scss';
+
+class ProductContainer extends Component {
+  render() {
+    return (
+      <main className="productContainer">
+        <img
+          alt="상품사진"
+          className="productImage"
+          src="./images/unsplash_bed.jpg"
```

Comment:
```suggestion
          src="/images/unsplash_bed.jpg"
```
절대경로로 적어주세요! 상대경로로 지정하게 되면 path의 depth가 깊어졌을때 원하는대로 public 폴더에 접근을 하지 못합니다!
예시) 상대경로로 작성했을 시
- http://localhost:3000/main => 이경우에는 상대경로로 작성해도 public 폴더로 정상 인식해서 정상동작
- http://localhost:3000/detail/1 => 이경우에는 path의 depth가 깊어져서 상대경로로 지정하게되면 pubilc/detail 폴더안에서 찾으려고 하기 때문에 제대로 동작하지 않음.